elastic/kibana
# [maps] display vector tile results in vector tile inspector

Closes https://github.com/elastic/kibana/issues/172717

PR updates Vector tile inspector to display vector tile response. When tile has an error, error displays "View details" button to open response in inspector.

### Test setup
1. install sample web logs data set
2. create new map
3. add heatmap layer. 
4. Save map

### Test response displayed in inspector
1. Click "Inspector" button in top nav
2. Click "Response" tab. Verify response is displayed
    <img width="500" alt="Screenshot 2023-12-05 at 1 28 41 PM" src="https://github.com/elastic/kibana/assets/373691/a90ea2bf-92c5-45aa-b98a-3edf45c3d9fa">

### Vector tile error contains "View details" button that opens inspector
1. add filter
    ```
    {
      "error_query": {
        "indices": [
          {
            "error_type": "exception",
            "message": "local shard failure message 123",
            "name": "kibana_sample_data_logs"
          }
        ]
      }
    }
    ```
    <img width="250" alt="Screenshot 2023-12-05 at 1 29 27 PM" src="https://github.com/elastic/kibana/assets/373691/10fa5341-6901-477f-a2dc-b77bb2891ae0">
5. Click "View details". Verify inspector is showing response for correct layer and tile
    <img width="500" alt="Screenshot 2023-12-05 at 1 29 33 PM" src="https://github.com/elastic/kibana/assets/373691/2b78a865-44e9-42c2-9e61-f09e7491e17f">
